README.md

```diff
@@ -72,6 +72,94 @@ yum install tcpdump
 apt-get install tcpdump
 ```
 
+### Using `bf_pktpy` as an alternate packet manipulation module
+
+The Python module `bf_pktpy` is included as part of the
```

Comment:
We should consider making that module standalone or upstreaming it to PTF.

Reply:
I have a temporary repo with only the bf_pkpy code by itself here, and it does allow one to pip install it just fine in my testing: https://github.com/jafingerhut/bf-pktpy

I don't have a big opinion on whether open-p4studio should be modified to install it from a new https://github.com/p4lang/bf-pktpy repo, or if we end up with 2 copies of it.

Reply:
I think it is easiest to vendor it as part of ptf, because it is specially built for this use case.

Reply:
Yes, partially for convenience. Partially because bf_pktpy is specialized towards PTF and this coupling makes sure its functionality is scoped for PTF-style use cases.